JamJar/JamJarEdge.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  JamJarEdge.swift
+//  JamJar
+//
+//  Created by Ethan Riback on 4/2/16.
+//  Copyright © 2016 JamJar. All rights reserved.
+//
+
+import ObjectMapper
+
+class JamJarEdge: NSObject, Mappable {
+
+    // JamJarEdge Attributes
+    var confidence: Int!
+    var offset: Double!
+    var video: Int!
```

Comment:
For sake of naming clarity we should probably call this var `videoId` since it's not the actual video itself